### PR TITLE
Smusand2/readme

### PR DESCRIPTION
**Pull request by samsanders23**
Small changes, just the readme (with a cost breakdown), + reducing the size of the Aurora write node for cost purposes.

Update: also several security alterations, in line with a Security Review.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cfn/rds.template [hunk 1]
```diff
@@ -71,7 +71,7 @@ Resources:
   DBWriterInstance:
     Type: AWS::RDS::DBInstance
     Properties:
-      DBInstanceClass: !Ref DBInstanceClass
+      DBInstanceClass: db.t3.small
```
Comment (rezabekf) [inline cfn/rds.template]: This is not a right place to define instance class. This proposed change will hard code the value, make it difficult in a future to change the size. If you need to update the size, please change the default value here https://github.com/aws-samples/amazon-rds-proxy-multitenant-load-test/blob/613bf844bcee3310d15c1cdb8a9c27bb2f4d2c6e/cfn/main.template#L87

**Review by rezabekf — CHANGES_REQUESTED** [review]
See the inline comment for `rds.template`

File: cfn/rds.template (hunk 1 above)
Comment (rezabekf) [reply cfn/rds.template]: If you need to introduce different sizes for "Reader" and "Writer" nodes, I would recommend to do so in a same fashion it is done now; as a parameter passed from main stack down to rds stack.

Comment (samsanders23) [reply cfn/rds.template]: Done

**Comment by samsanders23** [comment]
All changes fixed, please approve for merge

File: Makefile [hunk 2]
```diff
@@ -38,6 +38,7 @@ deploy: package
 	  --capabilities CAPABILITY_NAMED_IAM CAPABILITY_AUTO_EXPAND \
 	  --parameter-overrides \
 	  	CreateLoadTest=$(LOAD_TEST) \
+		CIDRRange=$(CIDR_RANGE) \
```
Comment (rezabekf) [inline Makefile:41]: Please add any overrides to `.custom.mk.example` too as this is used to define your parameters.

File: README.md [hunk 3]
```diff
@@ -54,15 +54,16 @@ template are pre-populated. Click the *Next* button at the bottom of the page.
 |---------------|-------|-----------|
 |CreateLoadTest|true|If True, this creates a Load Test VPC and an accompanying No Proxy VPC, in order to run a load test and compare metrics between Proxy and No Proxy.|
 |AvailabilityZones|Requires input|The list of Availability Zones to use for the subnets in the VPC. Select two Availability Zones from the list.|
-|DBInstanceClass|db.r5.large|The database instance class for the Proxy and No Proxy VPC Amazon Aurora instances, for example db.m5.large.|
+|DBWriterClass|db.t3.small|The database instance class for the Proxy and No Proxy VPC Amazon Aurora Writer, for example db.m5.large.|
```
Comment (rezabekf) [inline README.md]: Please change "for example" to "db.t3.small" to reflect the default value.

File: cfn/access-function.template [hunk 4]
```diff
@@ -25,7 +25,7 @@ Parameters:
     Type: AWS::EC2::SecurityGroup::Id
 
   ProxySecurityGroup:
-    Type: String # Has to be type String as NoProxy has 'NA' value
```
Comment (rezabekf) [inline cfn/access-function.template:28]: Why was this removed? It is rather important comment if anyone wants to modify template. But yeah, can be removed if you think it is not needed.

**Review by rezabekf — CHANGES_REQUESTED** [review]

File: cfn/access-function.template (hunk 4 above)
Comment (samsanders23) [reply cfn/access-function.template:28]: Nick requested for me to remove this and all other comments in this template

**Comment by samsanders23** [comment]
All changes implemented / commented on

File: README.md [hunk 5]
```diff
@@ -54,15 +54,16 @@ template are pre-populated. Click the *Next* button at the bottom of the page.
 |---------------|-------|-----------|
 |CreateLoadTest|true|If True, this creates a Load Test VPC and an accompanying No Proxy VPC, in order to run a load test and compare metrics between Proxy and No Proxy.|
 |AvailabilityZones|Requires input|The list of Availability Zones to use for the subnets in the VPC. Select two Availability Zones from the list.|
-|DBInstanceClass|db.r5.large|The database instance class for the Proxy and No Proxy VPC Amazon Aurora instances, for example db.m5.large.|
+|DBWriterClass|db.t3.medium|The database instance class for the Proxy and No Proxy VPC Amazon Aurora Writer, for example db.m5.large.|
```
Comment (rezabekf) [inline README.md]:
```suggestion
|DBWriterClass|db.t3.medium|The database instance class for the Proxy and No Proxy VPC Amazon Aurora Writer, for example db.t3.medium.|
```

**Comment by rezabekf** [comment]
> All changes implemented / commented on

There is still missing override parameter in `.custom.mk.example`. I will push the change myself to not go back and forth.

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**